src/config/display.c

```diff
@@ -342,5 +274,24 @@ u8 CONFIG_ReadDisplay()
     #else
     char filename[] = "media/config.ini";
     #endif
-    return CONFIG_IniParse(filename, ini_handler, (void *)&Display);
+    if (CONFIG_IniParse(filename, ini_handler, (void *)&Display)) {
+        for (int i = 0; i < NUM_LABELS; i++) {
+            struct LabelDesc *label = &Display.font[i];
```

Comment:
can you throw this into a 'legacy_hacks' function?  it will make me feel better to have the legacy cruft grouped together in its own function rather than inline